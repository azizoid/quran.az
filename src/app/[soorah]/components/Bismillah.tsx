import type { JSX } from 'react'
export const Bismillah = (): JSX.Element => (
  <li className="ayah-list-item text-center font-Nunito text-4xl">&#65021;</li>
)
