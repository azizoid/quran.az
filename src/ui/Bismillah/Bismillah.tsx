export const Bismillah = (): JSX.Element => (
  <div className="ayah-list-item text-center text-4xl font-Nunito">&#65021;</div>
)
