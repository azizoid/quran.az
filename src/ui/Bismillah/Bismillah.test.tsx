import { render } from '@testing-library/react'
import { Bismillah } from './Bismillah'

test('Render Bismillah', () => {
  const { container } = render(<Bismillah />)

  expect(container).toMatchSnapshot()
})
