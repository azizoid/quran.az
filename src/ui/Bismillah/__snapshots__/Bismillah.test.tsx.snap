// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Bismillah 1`] = `
<div>
  <div
    class="ayah-list-item text-center text-4xl font-Nunito"
  >
    ï·½
  </div>
</div>
`;
